src/main/java/emissary/command/RunCommand.java

```diff
@@ -54,7 +54,7 @@ public void run(JCommander jc) {
         } catch (ClassNotFoundException e) {
             throw new RuntimeException("Could not find fully qualified class named " + clazzName);
         } catch (IllegalAccessException | IllegalArgumentException | InvocationTargetException | NoSuchMethodException | SecurityException e) {
-            e.printStackTrace();
+            LOG.error("Problem calling main from " + clazzName + " with " + Arrays.toString(clazzArgs), e);
```

Comment:
In situations like these, a pattern we've used before is to build a string once and then send that string to both the log and pass it to the exception.